R/plumber.R

```diff
@@ -817,7 +817,8 @@ Plumber <- R6Class(
       private$docs_info <- list(
         enabled = enabled,
         docs = docs,
-        args = list(...)
+        args = list(...),
+        has_not_been_set = FALSE
```

Comment:
Saw that this was set during router initialization, I think it should be set to FALSE when `setDocs` is called. Otherwise it stays TRUE at all time.

Reply:
I'm good with reversing the name / logic. Ex: ~ `vanilla = TRUE`

Reply:
Let me rephrase that. `has_not_been_set` is not set anywhere else than initialization. So the value will always be what you set it to when you check it later for the purpose of lifecycle messaging. This boolean value should be reversed when you actually set the doc. This is what I was alluding to.

Reply:
I'm up for changing the name / logic as long as we can tell the difference between the initialization and customized `docs_info`.  (I know my current implementation was a lil bit of a hack.)